src/assets/stylesheets/index.scss

```diff
@@ -298,6 +299,7 @@ html {
 
 body {
   font-family: var(--bs-body-font-family);
```

Comment:
With other font-families we ususaly add a `, sans-serif;`. Is this useful to add it for all font families?

Reply:
Yes. But it depends on the used font. Maybe you choose an serif font, then the fallback would be 'serif'.